cmds/tui.c

```diff
@@ -1121,16 +1181,16 @@ static void print_graph_indent(struct tui_graph *graph,
 
 	for (i = 0; i < depth; i++) {
 		if (!graph->disp_mask[i]) {
-			printw("   ");
+			println("   ");
 			continue;
 		}
 
 		if (i < depth - 1 || single_child)
-			printw("  │");
+			println("  │");
 		else if (is_last_child(parent, node))
-			printw("  └");
+			println("  └");
 		else
-			printw("  ├");
+			println("  ├");
```

Comment:
It'd be simpler if we could return number of characters printed and account each special case.